scripts/align_events_and_rules.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python
+import argparse
+import datetime
+from dateutil import tz
+import requests
+from os import abort
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Align collection exercise events and rules')
+    parser.add_argument("url", help="Collection exercise service URL")
```

Comment:
we store this as scheme (e.g. http), host and port in the environment variables, so it might be worth passing these parameters individually, and then you wouldn't have to escape the slashes etc

Reply:
Just checked the devtest cf environments and the response-operations-ui config.py file, it looks like we have a `COLLECTION_EXERCISE_URL` environment variable so no need to pass them separately.